test/metabase/pulse_test.clj

```diff
@@ -1,16 +1,22 @@
 (ns metabase.pulse-test
-  (:require [expectations :refer :all]
+  (:require [clojure.walk :as walk]
```

Comment:
I first started working here and I think that test failures here will be more difficult to diagnose than in other tests that make use of the utilities I added to the fake-inbox stuff. I'll still need something like below for slack messages, but I'm having some second thoughts about mocking the send here instead of using the fake-inbox (maybe revisiting [this comment](https://github.com/metabase/metabase/pull/6268#pullrequestreview-71966517))